go.mod

```diff
@@ -1,19 +1,21 @@
 module github.com/kubestellar/kubestellar
 
-go 1.22
+go 1.22.7
+
+toolchain go1.23.2
```

Comment:
Just want to understand this. Why do we need a toolchain version higher than the one in `go 1.22.7`?

Reply:
I have the same question

Reply:
Oops, I thought that I had removed that. Fixed now.